templates/default/customer/customereditbox.html

```diff
@@ -128,17 +128,32 @@
 			<div class="lmsbox-panel">
 				<table>
 					<tr>
+						<td>
+							<label for="customerdata[extid]" class="nobr bold">
```

Comment:
`for` działa na `name` innego elementu? Raczej działa na `id` innego elementu.

Reply:
https://github.com/chilek/lms/pull/2308#discussion_r947766896